docs/02-usage/050_configuration.md

```
@@ -94,21 +115,36 @@ Examples of built-in modes include:
 
 Find the concrete definitions of these modes [here](https://github.com/oraios/serena/tree/main/src/serena/resources/config/modes).
 
+Active modes are configured in (from lowest to highest precedence):
+  * the global configuration file (`serena_config.yml`)
```

markdownlint-cli2 (0.20.0)

[warning] Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)